README.md

```diff
@@ -50,9 +50,21 @@ Development uses [nodemon](https://github.com/remy/nodemon) which automatically
 
 Visit [http://localhost:4000/](http://localhost:4000/) where you will be able to perform queries using [GraphQL Playground](https://github.com/graphql/graphql-playground).
 
+## Linting
+
+This repo uses the prettier for Linting, to check it you can use this command:
+```bash
+yarn prettier-check
```

Comment:
does this and the `.editorconfig` do the same formatting? or will the two result in conflicting formatting?

Reply:
They are different, we can probably remove `editorconfig` later, which is tied to your IDE based on what Parvez told me today.

Reply:
Oh, they are doing the same formatting, we tried to match them.

Reply:
cool. let's keep the editor config if it is doing the same thing as the linter